gitlab/v4/cli.py

```diff
@@ -53,7 +54,10 @@ def __init__(
         # the class _path attribute, and replace the value with the result.
         if TYPE_CHECKING:
             assert self.mgr_cls._path is not None
-        self.mgr_cls._path = self.mgr_cls._path.format(**self.args)
+
+        self._process_from_parent_attributes()
```

Comment:
suggestion: should we call it like the variable it processes? It will also keep it shorter then.
```suggestion
        self._process_from_parent_args()
```

Reply:
Done. Though I used `self._process_from_parent_attrs` as the variable is `_from_parent_attrs`

Reply:
Ah sorry, was looking at the `self.parent_args` above, thanks!